packages/storybook-ui-components/package.json

```diff
@@ -1,11 +1,11 @@
 {
   "name": "storybook-ui-components",
-  "version": "0.2.0",
+  "version": "0.2.5",
   "description": "Storybook supporting ui-components as a component catalog",
   "main": "index.js",
   "private": true,
   "scripts": {
-    "build": "build-storybook -c .storybook -o public",
+    "build-storybook": "build-storybook -c .storybook -o public",
```

Comment:
Why was this changed? I thought we wanted all of our packages to use the same script names?

Reply:
working on long description and back story in the PR description. Stay tuned.

Reply:
>changing `build` to be `build-storybook` to see if we can avoid building
storybook when publishing

Oh I see. I guess we don't really publish the storybook package

Reply:
The TLDR in this case is, that storybook doesn't need to be built during development life cycle, only during deployment. So I think the inconsistency with other packages is appropriate here.